tools/extras/travis_script.sh

```diff
@@ -66,16 +66,24 @@ CCC=$(mtoken CXX "$CXX")
 #fi
 
 echo "Building tools..." [Time: $(date)]
-runvx cd tools
-runvx make -j$MAXPAR openfst "$CCC" CXXFLAGS="$CF" \
-      OPENFST_CONFIGURE="--disable-static --enable-shared --disable-bin --disable-dependency-tracking"
-cd ..
+# runvx cd tools
```

Comment:
I assume some of these changes should be reverted before we merge this...

Reply:
Of course.

The purpose was to show how to enable travis to be able to run shellcheck without having to sit through the entire CI process.